vortex-dtype/src/dtype.rs

```diff
@@ -140,6 +140,14 @@ impl DType {
             _ => None,
         }
     }
+
+    /// Get the `PType` if `self` is a `PType`, otherwise `None`
+    pub fn as_ptype(&self) -> Option<&PType> {
```

Comment:
We have always used PType::try_from thus far

Reply:
removed.